airbyte-integrations/connectors/source-gnews/source_gnews/gnews.yaml

```diff
@@ -1,5 +1,109 @@
 version: "0.1.0"
 
+schemas:
+  search_stream_schema:
+    "$schema": http://json-schema.org/draft-07/schema#
```

Comment:
do we need this line? I think it makes sense at the top level of a yaml file to specify the version of jsonschema to use to validate, but I doubt it does anything here

Reply:
Ah, that can be removed. Just kept it in so I could diff the output of `discover` before and after the schema was included inline in the manifest.

Reply:
@girarda just pushed the update removing these lines. Did you want to review the change?